src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -1062,6 +1062,7 @@ private async Task SendHeartBeat(string reason)
             try
             {
                 await TcpConnection.Stream.WriteAsync(Heartbeat, 0, Heartbeat.Length, CancellationTokenSource.Token);
+                await TcpConnection.Stream.FlushAsync();
```

Comment:
Do we need review other places as well?

Reply:
I checked all the places we use the compression stream, this is the one that doesn't flush that I found

Reply:
Great